### DISCUSSION TITLE
RPC failed with reason :nodedown

### DISCUSSION THREAD

**Discussion by kekkodance** [plausible/analytics#3279] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

I'm running docker compose and plausible, i tried migrating to 2.0 but running the data migration command results in an error:

`--rpc-eval : RPC failed :nodedown`

### Expected behavior

Migration is successful

### Environment

```markdown
- OS: Ubuntu Minimal 20.04 (Oracle Cloud) ARM64
```

**Comment by ruslandoga** [comment]
👋 @kekkodance 

https://github.com/plausible/analytics/discussions/2896#discussioncomment-5986941 is probably relevant: `RELEASE_DISTRIBUTION=none` should ***not*** be set for rpc to work.

**Comment by kekkodance** [comment]
Migration completed, tho i can't enable "all time" and the "Days" option at the same time?

**Reply by ruslandoga** [reply]
It might be something similar to https://github.com/plausible/analytics/issues/2982
